runtime/src/main/java/org/corfudb/protocols/wireprotocol/orchestrator/ActionStatus.java

```diff
@@ -1,4 +1,4 @@
-package org.corfudb.infrastructure.orchestrator;
+package org.corfudb.protocols.wireprotocol.orchestrator;
```

Comment:
is this the correct package. I am just asking as I am not sure of the scope of wireprotocol package.

Reply:
Runtime doesn't depend on infrastructure, so I can't move them to the orchestrator server because it would then depend on the runtime.